lib/force.go

```diff
@@ -1270,8 +1276,13 @@ func (f *Force) httpGetRequestAndSend(url string, headers map[string]string, res
 				Data:         data,
 			}
 		}
+
 		if err == io.EOF || err == io.ErrUnexpectedEOF {
-			return nil
+			if retryMode == retry && firstChunk && err == io.EOF {
+				f.httpGetRequestAndSend(url, headers, results, doNotRetry)
```

Comment:
The return value is being implicitly discarded.  Should it be this?
```suggestion
				return f.httpGetRequestAndSend(url, headers, results, doNotRetry)
```

Reply:
Good catch.